src/nbl/ui/CWindowWin32.cpp

```diff
@@ -485,6 +513,132 @@ namespace ui
 		return DefWindowProc(hWnd, message, wParam, lParam);
 	}
 
+	E_KEY_CODE CWindowWin32::getNablaKeyCodeFromNative(uint32_t nativeWindowsKeyCode)
+	{
+		nbl::ui::E_KEY_CODE nablaKeyCode = EKC_NONE;
+		switch (nativeWindowsKeyCode)
+		{
+		case VK_BACK:			nablaKeyCode = EKC_BACKSPACE; break;
+		case VK_TAB:			nablaKeyCode = EKC_TAB; break;
+		case VK_CLEAR:			nablaKeyCode = EKC_CLEAR; break;
+		case VK_RETURN:			nablaKeyCode = EKC_ENTER; break;
+		case VK_SHIFT:			[[fallthrough]];
+		case VK_LSHIFT:			[[fallthrough]];
+		case VK_RSHIFT:			nablaKeyCode = EKC_SHIFT; break;
+		case VK_CONTROL:		[[fallthrough]];
+		case VK_LCONTROL:		[[fallthrough]];
+		case VK_RCONTROL:		nablaKeyCode = EKC_CONTROL; break;
```

Comment:
same for control as for shift
https://github.com/Crisspl/Nabla/pull/5/commits/b75225f9d5b4e8cce14c891b36c1d0934504a53c?file-filters%5B%5D=.cpp#r644364976